clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/AggVersionedStorageEngineStats.java

```diff
@@ -32,6 +33,9 @@ public void setStorageEngine(String topicName, AbstractStorageEngine storageEngi
       LOGGER.warn("Invalid topic name: {}", topicName);
       return;
     }
+    if (VeniceSystemStoreUtils.isSystemStore(topicName)) {
```

Comment:
Why are we not emitting storage engine metrics for system stores? I believe their ingestion is still critical for Venice to function (I know we are trying to move away from system store but that's still the case today and there are some system stores that will fall under this check that are not going away any time soon).